README.md

```diff
@@ -72,7 +72,7 @@ At the center of every mini rack is a 10" or 'half-width' rack. A typical rack i
 | [Digitus Network Cabinet](https://amzn.to/3ZIjjBP) | 6U, 9U, 12U | wall-mountable |
 | [DIY Laser Cut Mini Rack](https://github.com/ssilverm/lasercut-mini-rack)|6U||
 | [DSIT Rack](https://www.redcorp.com/en/product/servers-racks-options/dsit/server-rack-15u-glass-door-black-312x310x752mm-wxdxh--ds10-3315/y3872053#) | 15U | glass door |
-| [Flyht Pro Stage Rack Double Door](https://www.thomann.de/intl/flyht_pro_stage_rack_95_6u_double_door.htm) | 6U | flight case, similar models also available down to 2U |
+| [Flyht Pro Stage Rack Double Door](https://www.thomann.de/intl/flyht_pro_stage_rack_95_6u_double_door.htm) | 6U | flight case, similar models also available down to 2U, requires modification to fit 10" rack equipment. Rails are spaced 20mm farther apart from the [de facto standard](#standard). |
```

Comment:
Maybe just `requires [slight modification](https://github.com/geerlingguy/mini-rack/issues/99)` to keep it short in the table?

And then that issue can be the canonical source of info about the mods required.

Reply:
Sure thing. I'll update it shortly.